src/components/MarkdownProvider/components/Anchor.ts

```diff
@@ -5,37 +5,30 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled, { DefaultTheme } from 'styled-components';
-import { getColor, mediaQuery } from '@zendeskgarden/react-theming';
+import styled from 'styled-components';
+import { SELECTOR_FOCUS_VISIBLE, mediaQuery } from '@zendeskgarden/react-theming';
 import { Anchor } from '@zendeskgarden/react-buttons';
-import { math, rgba } from 'polished';
-
-const boxShadow = (theme: DefaultTheme) => {
-  const color = getColor('primaryHue', 600, theme);
-  const shadow = rgba(color as string, 0.35);
-
-  return `inset ${theme.shadows.sm(shadow)}`;
-};
+import { math } from 'polished';
 
 export const StyledAnchor = styled(Anchor).attrs(props => ({
   /* eslint-disable-next-line prefer-named-capture-group */
   isExternal: props.href && !/^(#|\/(?!\/))/u.test(props.href)
 }))`
   &.anchor.before {
     position: relative;
-    margin-left: -${p => math(`${p.theme.space.xs} * 2 + ${p.theme.iconSizes.md}`)};
+    margin-right: ${p => p.theme.space.base}px;
+    margin-left: -${p => math(`${p.theme.space.xs} * 2 + ${p.theme.iconSizes.md} + ${p.theme.space.base}`)};
     border-radius: ${p => p.theme.borderRadii.md};
     padding: 0 ${p => p.theme.iconSizes.md};
     color: transparent;
 
     /* stylelint-disable selector-max-specificity */
     ${p => mediaQuery('down', 'md', p.theme)} {
-      margin-left: -${p => math(`${p.theme.space.xxs} * 2 + ${p.theme.iconSizes.md}`)};
+      margin-left: -${p => math(`${p.theme.space.xxs} * 2 + ${p.theme.iconSizes.md} + ${p.theme.space.base}`)};
```

Comment:
Why were these margins modified? The addition of `theme.space.base` doesn't appear to be based on context logic.

Reply:
These margins were modified to account for `outline` overlap of the SVG. Compare with and without:
![without](https://github.com/zendeskgarden/website/assets/12474067/3bf1d349-1e8c-46fc-8e57-6d653f121f14)
![with](https://github.com/zendeskgarden/website/assets/12474067/1438ea35-5111-4415-a7e3-9299a84a2148)

Reply:
Using the `theme.borderWidth.md` to position has the same effect as without (from above). The use of `margin-right` positions the headings in their current position while moving the anchor further out to avoid overlap of the focus ring.
![border-width](https://github.com/zendeskgarden/website/assets/12474067/c7cc8a71-60f5-47d3-b282-068ce65b1a65)

Reply:
I see. Let's override (remove) the outline styling here, use `box-shadow` instead (as these are not wrap-able text strings; but more like buttons visually), and revert messing with the margining. The current layout isn't spaced as originally intended.

Reply:
The only part that is moving in the current layout is the anchor. Special attention was given to keep the title text in the original placement (the reason for the `margin-right` usage). Is the anchor shifting what's concerning or the title shifting?

Reply:
The outline rings here are larger than what we see with the current website's use of inset `box-shadow` and we know from past design passes that this is not desired for mobile. Please override to stick with the original `box-shadow: inset ...` plan.

Reply:
Just updated!